packages/scripts/scripts/build.js

```diff
@@ -28,8 +29,14 @@ if ( hasWebpackConfig ) {
 	);
 	process.exit( status );
 } else {
-	// eslint-disable-next-line no-console
-	console.log( 'Webpack config file is missing.' );
-	process.exit( 1 );
+	// Sets environment to production.
+	process.env.NODE_ENV = 'production';
+
+	const { status } = spawn(
+		resolveBin( 'webpack' ),
+		[ '--config', fromConfigRoot( 'webpack.config.js' ) ],
```

Comment:
It seems like currently, we could consolidate those 2 if statement branches into one and provide a different set of arguments depending on the check whether Webpack config exists.

Reply:
I've already done that!